mllib/src/main/scala/org/apache/spark/ml/tree/impl/TreePoint.scala

```diff
@@ -125,7 +127,7 @@ private[spark] object TreePoint {
           s"DecisionTree given invalid data:" +
             s" Feature $featureIndex is categorical with values in {0,...,${featureArity - 1}," +
             s" but a data point gives it value $featureValue.\n" +
-            "  Bad data point: " + labeledPoint.toString)
+            "  Bad data point: " + instance.toString)
```

Comment:
Total nit, but this can be `s"...: $instance"`

Reply:
done!